docs/usage.md

```diff
@@ -466,40 +664,43 @@ information on the exact syntax.
 Please find a [Shell Script](script_update.md) that can be used
 in a Jenkins or any other CI engine to report new bugs.
 
-
 #### Alternative 2: Store each analysis in a new run <a name="storing-new-runs"></a>
-
 Each daily analysis should be stored as a new run name, for example using the
 following naming convention: `<module_name>_<branch_name>_<date>`.
 
 Using `tmux` with daily analysis as example:
 
 1. Generate a new log file
 ```sh
-CodeChecker log -b "make" -o compilation.json
+CodeChecker log --build "make" --output compile_commands.json
 ```
 2. Re-analyze the project. Make sure you use the same analyzer options all the
    time, as changing enabled checkers or fine-tuning the analyzers *may*
    result in new bugs being found.
 ```sh
-CodeChecker analyze compilation.json -o ./reports-daily
+CodeChecker analyze ./compile_commands.json --output ./reports-daily
 ```
 3. Store the analysis results into the central CodeChecher server
 ```sh
-CodeChecker store ./reports --url http://localhost:8555/Default --name tmux_master_$(date +"%Y_%m_%d")
+CodeChecker store ./reports --url http://localhost:8555/Default \
+    --name tmux_master_$(date +"%Y_%m_%d")
```

Comment:
I thought that we are using our own example project to demonstrate the usage of CodeChecker. So why do we reference tmux in this section?

Reply:
Because this example has not a real repo and branches. While this section is hipothetical because gerrit integration is not a deal for everybody, I left this section untouched.